### PR TITLE
Adds medibeam to all Syndicate modes.[READY][TESTED] 

### PR DESCRIPTION
**Pull request by ghost**
[Changelogs].

:cl: lord saladin(ckey)
add: Adds medibeam gun to traitor uplinks.
Tweak: Now costs 10 TC again(if you die to two traitors using medibeam guns then it's your fault.) and has a chance to spawn in surplus and null crates, Making it more inline with Traitor modes, restricted to MD/CMO otherwise.
Update: now has correct path.
/:cl:

[why]: Want to feel like your a Odysseus or be a friendly traitor, now you can as this pr adds the medibeam gun to uplinks. NOTE yorii has tested and it works.

### CODE REVIEW

**Comment by GuppyLaxx** [comment]
Let me stop you right there and say........ **Friendly traitors are gay.**
Apart from that, 👍 I'm expecting some funny gimmicks

**Comment by Jarektheraptor** [comment]
Time to cross the beams

**Comment by lordsaladin** [comment]
Because every game needs a ranged medic.

**Comment by Cyprex** [comment]
I suggest just removing the gamemode requirement instead.

**Comment by GuppyLaxx** [comment]
10 TC sounds like 2 greytides rushing in and medibeaming the captain at the same time

**Comment by CitrusGender** [comment]
isn't this what you just did in your last PR that got closed except you're adding it to traitor mode

**Comment by lordsaladin** [comment]
@CitrusGender  This grew out of that idea, less memey now.
Was told to make a separate pr for this.

**Comment by CitrusGender** [comment]
ok now it's just in all modes

aight

**Comment by Cyprex** [comment]
Their previous PR changed various surplus chances for null crates.

**Comment by lordsaladin** [comment]
Well it's intent was to make sure you get trolled by rng less,

**Comment by CitrusGender** [comment]
yeah the surplus chance is changed in this one too for just medibeams now (in addition to the other Prs)

**Comment by lordsaladin** [comment]
@CitrusGender  the last one was closed.

**Comment by GuppyLaxx** [comment]
last one changed surplus values of a ton of shit WHILE adding medbeams to traitors

**Comment by lordsaladin** [comment]
@Spaceinaba  They overproduced them and they want to encouraged team work,Also blood brothers exist.

**Comment by Spaceinaba** [comment]
Blood brothers don't have TC last I checked.

**Comment by lordsaladin** [comment]
@Spaceinaba Ok they wanted to encourage betrayal,Also y'all keep saying be the change the world needs so this is that change.(plus sec will give it to medical if you are caught with it as it is clearly not a weapon.)

**Comment by PKPenguin321** [comment]
friendly traitors suuuuck

**Comment by lordsaladin** [comment]
@PKPenguin321 Have you ever played a role where having a friendly traitor is handy, or are you a greytide main?

**Comment by Wilchenx** [comment]
Medbeams cost 10 tc now


Can't wait to see traitor duos cross beams on people in hilarious murder spree.

**Comment by lordsaladin** [comment]
That takes more teamwork then le desword no slip muderbones.

**Comment by davethwave** [comment]
Might make more sense to make it a medical only item.

**Comment by lordsaladin** [comment]
@davethwave  nah then it would be another job specific item and i don't want that.

**Comment by PKPenguin321** [comment]
@lordsaladin fuck you, "if you disagree with me u r le greytide main." antagonists are there to provide conflict in the round and make it actually fucking meaningful rather than just the crew grinding out all the same research/botany/etc that it did the same way the round before. traitors that are explicitly "friendly" and go out of their way to not only not antagonize but stop other antagonists actively disrupt the conflict of the round and make the game boring as fuck. if your only comeback is that i'm a """greytide main""" (i'm not, and what the fuck does that even have to do with anything, you absolute retard), you can fuck yourself.

**Comment by Bawhoppen** [comment]
Absolutely no reason for this change. An OP boring item like this that only makes sense in the context of a team mode like nukeops definitely shouldn't be added to standard traitor. 10TC is also way way too cheap.
At best, you might find it reasonable to add as a job specific traitor item for doctors, otherwise this is a big fat :-1: from me.

**Comment by lordsaladin** [comment]
@PKPenguin321 wow iirc Github have explicitly told us not to swear on here in the past,Have you not played terry or lowpop rounds if the antag goes SSD or dies early it basically becomes extended and we can get stuff done, Don't forget betrayal exists you can just cause mayhem when they least expect it.

**Comment by lordsaladin** [comment]
THERE YOU UNGRATEFUL BEEPS NOW IT COSTS ALL YOUR TC.

**Comment by GLACoding** [comment]
Can you remove the surplus chance?
Its a pretty costly and extremely niche item that would basically be useless for a normal traitor which would dilute the overall value of his surplus drop if he was unlucky enough to get it.

**Comment by lordsaladin** [comment]
@GLACoding ok then i'll remove all the other  restricted items from surplus crates no more brain washing  or loyal borgs or you!

**Comment by CitrusGender** [comment]
here we go

**Comment by lordsaladin** [comment]
All this burning my eyes on github's website trying to implement a idea for y'all to salt on it,you wouldn't like it if i did that to your prs.

**Comment by CitrusGender** [comment]
tweaks/balances get scrutiny since they're all just opinions. This isn't new.

**Comment by optimumtact** [comment]
if you are this sensitive about people giving you critical feedback you will not last very long here at all.

Try to seperate peoples comments on your change with their comments on you as a person or you will quickly burn out

**Comment by ExcessiveUseOfCobblestone** [comment]
That and stop making a diff pull request lol

**Comment by kevinz000** [comment]
is this tested

**Comment by Tlaltecuhtli** [comment]
make the medibeam have a second mode where it does negative healing

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -1149,9 +1149,10 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	desc = "A wonder of Syndicate engineering, the Medbeam gun, or Medi-Gun enables a medic to keep his fellow \
 			operatives in the fight, even while under fire."
```
Comment (Tlaltecuhtli) [inline code/modules/uplink/uplink_items.dm]: operatives? more like metafriends

**Review by Tlaltecuhtli — CHANGES_REQUESTED** [review]
also look at the end of file the restricted roles items have a different path

**Comment by Iatots** [comment]
Are you from terry?

**Comment by lordsaladin** [comment]
Terry. yes
update Yorii has tested and it works.

**Comment by Noahnoah55** [comment]
I wonder if you can use a gun firing circuit with the medigun. Sounds like it could get very broken very quickly.

File: code/modules/uplink/uplink_items.dm [hunk 2]
```diff
@@ -1147,11 +1147,12 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 /datum/uplink_item/device_tools/medgun
```
Comment (Tlaltecuhtli) [inline code/modules/uplink/uplink_items.dm]: needs to be 
/datum/uplink_item/role_restricted/medgun

**Review by Tlaltecuhtli — CHANGES_REQUESTED** [review]

**Comment by epochayur** [comment]
you can't go making random balance changes for no apparent reason and not expect to get some backlash

**Comment by improvedname** [comment]
I guess i can get syndicate borgs and medibeams trough null crates noe as a non antag

**Comment by iskyp** [comment]
what @epochayur said except you're not doing this for kicks

**Comment by AlexTheSysop42** [comment]
@improvedname first, null crates give op gear during op rounds so maybe

Second, non-ops can't use syndie spawners

**Comment by ExcessiveUseOfCobblestone** [comment]
@AlexTheSysop42 he's changing the medbeam to be not op specific, so you'd be able to get medbeams in nulls and surplus post this change (unless he sets `surplus_nullcrates` to 0).

**Comment by AlexTheSysop42** [comment]
He specifically mentioned syndicate borgs

**Comment by improvedname** [comment]
syndicate borgs is how you get a emagg as a non antag

**Review by ghost — COMMENTED** [review]
Everything checks out.

**Comment by lordsaladin** [comment]
Oh wow several admins dislike this?
Geez let me have my 2 minutes of fame on the changelog.

**Comment by iskyp** [comment]
@lordsaladin this is not how you get people to agree with your point

**Comment by epochayur** [comment]
untitled
jesus christ

**Comment by iskyp** [comment]
\> +1's their own PR
hahahahahahahahahahahahahahahahahahahahahahahahahahaha

**Comment by davethwave** [comment]
It would be interesting if you can at least get medibeams in null crates

**Comment by lordsaladin** [comment]
@davethewave this adds that.

**Comment by ExcessiveUseOfCobblestone** [comment]
@lordsaladin just fyi admins have 0 power here, and headmins only have say out of communal interest between admins/code (features would be pretty useless if you just got banned for using them 🤔).

However it seems like a nice bit of players dislike the PR as well.

**Comment by lordsaladin** [comment]
Of course they do this involves surplus/null crates.

**Comment by ShizCalev** [comment]
image

**Comment by lordsaladin** [comment]
@ShizCalev That not a valid reason or are you apart of the surplus/null crate hate cabal, like the rest of the people that disliked that want to deny me of having my name on the changelog.

**Comment by PKPenguin321** [comment]
You need to not take things personally and to stop acting so entitled

**Comment by iskyp** [comment]
@lordsaladin its very easy to get your name on the channeling, just make a fix. you're not entitled to anything just because you think your changes are better for the game

also, I'm betting an entire bag of popcorn that half of the people who -1d this probably did it because you had a temper tantrum whenever you were told this was a bad change

**Comment by ExcessiveUseOfCobblestone** [comment]
📸 ADD 📸 ME 📸 TO 📸 THE 📸  SCREENSHOT 📸

**Comment by Armhulen** [comment]
:camera_flash: Yep, this one's going into my usual github activity compilation.

**Comment by CitrusGender** [comment]
based shiz

**Pull request closed without merging**